test/e2e/apimachinery/storage_version.go

```diff
@@ -34,7 +34,8 @@ const (
 	idNonExist = "id-non-exist"
 )
 
-var _ = SIGDescribe("StorageVersion resources [Feature:StorageVersionAPI]", func() {
+// This test requires that --feature-gates=APIServerIdentity=true,StorageVersionAPI=true be set on the apiserver and the controller manager
+var _ = SIGDescribe("StorageVersion resources [Feature:StorageVersionAPI, APIServerIdentity]", func() {
```

Comment:
changing this tag will break the alpha e2e job config at https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L413